tests/e2e/login/privacy-policy.spec.js

```diff
@@ -76,109 +74,122 @@ describe('Privacy policy', () => {
     }
   };
 
-  const closeTourModal = () => {
-    helper.waitElementToBeVisible(element(by.css('.modal-dialog .modal-footer .btn')));
-    element(by.css('.modal-dialog .modal-footer .btn')).click();
+  const closeTourModal = async () => {
+    await helper.waitElementToBeVisibleNative(element(by.css('.modal-dialog .modal-footer .btn')));
+    await element(by.css('.modal-dialog .modal-footer .btn')).click();
   };
 
-  beforeAll(() => utils.saveDocs([privacyPolicies, PARENT_PLACE]));
-  afterEach(() => utils.revertDb(['privacy-policies', 'PARENT_PLACE']));
+  beforeAll(async () => {
+    await utils.saveDocsNative([privacyPolicies, PARENT_PLACE]);
+  });
+
+  afterEach(async () => {
+    await utils.revertDbNative(['privacy-policies', 'PARENT_PLACE']);
+  });
 
-  afterAll(() => {
-    commonElements.goToLoginPage();
-    loginPage.login(auth.username, auth.password);
-    return utils.revertDb();
+  afterAll(async () => {
+    await commonElements.goToLoginPageNative();
+    await loginPage.loginNative(auth.username, auth.password);
+    await utils.revertDbNative();
   });
 
   describe('for an online user', () => {
-    afterEach(() => utils.deleteUsers([onlineUser]));
+    afterEach(async () => {
+      await utils.deleteUsersNative([onlineUser]);
+    });
 
-    it('should show the correct privacy policy on login', () => {
-      browser.wait(() => utils.createUsers([onlineUser]).then(() => true));
-      commonElements.goToLoginPage();
-      loginPage.login('online', password);
+    it('should show the correct privacy policy on login', async () => {
+      await utils.createUsersNative([onlineUser]);
 
-      helper.waitElementToPresent(element(by.css('#privacy-policy-wrapper')));
+      // After first login, check that privacy policy was prompted to user
+      await commonElements.goToLoginPageNative();
+      await loginPage.loginNative('online', password);
+      await helper.waitElementToPresentNative(element(by.css('#privacy-policy-wrapper')));
```

Comment:
Can you move the elements to a page object? I think they would work in the `login.po.js`

Reply:
After taking look, I decided to move this into a new page object, privacy-policy.po since privacy policy was developed as an independent page and not closely related to login.